site2/docs/getting-started-standalone.md

```diff
@@ -1,278 +1,150 @@
 
-Once you have an up-to-date local copy of the release, you can start a local cluster using the [`pulsar`](reference-cli-tools.md#pulsar) command, which is stored in the `bin` directory, and specifying that you want to start Pulsar in standalone mode.
+Run this command to produce a message:
 
 ```bash
-bin/pulsar standalone
+bin/pulsar-client produce quickstart --messages 'Hello Pulsar!'
 ```
 
-If you have started Pulsar successfully, you will see `INFO`-level log messages like this:
+## Read messages from the topic
 
-```bash
-21:59:29.327 [DLM-/stream/storage-OrderedScheduler-3-0] INFO  org.apache.bookkeeper.stream.storage.impl.sc.StorageContainerImpl - Successfully started storage container (0).
-21:59:34.576 [main] INFO  org.apache.pulsar.broker.authentication.AuthenticationService - Authentication is disabled
-21:59:34.576 [main] INFO  org.apache.pulsar.websocket.WebSocketService - Pulsar WebSocket Service started
-```
+Now that we've written message to the topic, we'll read those messages back.
 
-:::tip
-
-* The service is running on your terminal, which is under your direct control. If you need to run other commands, open a new terminal window. 
-* To run the service as a background process, you can use the `bin/pulsar-daemon start standalone` command. For more information, see [pulsar-daemon](/docs/en/reference-cli-tools/#pulsar-daemon).
-* To perform a health check, you can use the `bin/pulsar-admin brokers healthcheck` command. For more information, see [Pulsar-admin docs](/tools/pulsar-admin/).
-* When you start a local standalone cluster, a `public/default` [namespace](concepts-messaging.md#namespaces) is created automatically. The namespace is used for development purposes. All Pulsar topics are managed within namespaces. For more information, see [Topics](concepts-messaging.md#topics).
-* By default, there is no encryption, authentication, or authorization configured. Apache Pulsar can be accessed from a remote server without any authorization. See [Security Overview](security-overview.md) for how to secure your deployment. 
-
-:::
-
-## Use Pulsar standalone
-
-Pulsar provides a CLI tool called [`pulsar-client`](reference-cli-tools.md#pulsar-client). The pulsar-client tool enables you to consume and produce messages to a Pulsar topic in a running cluster. 
-
-### Consume a message
-
-The following command consumes a message with the subscription name `first-subscription` to the `my-topic` topic:
+Run this command to launch the consumer:
 
 ```bash
-bin/pulsar-client consume my-topic -s "first-subscription"
+bin/pulsar-client consume quickstart -s 'first-subscription' -p Earliest -n 0
 ```
 
-If the message has been successfully consumed, you will see a confirmation like the following in the `pulsar-client` logs:
+Earliest means consuming from the earliest **unconsumed** message. `-n` configures the number of messages to consume, 0 means to consume forever.
```

Comment:
Suggest adding the link https://pulsar.apache.org/tools/pulsar-client/ for more reference

Reply:
Current https://pulsar.apache.org/tools/pulsar-client/ doesn't have a good landing page. Users can be confused if they're redirected to such a page:

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/188557570-bc1fcb91-f96b-4b27-bd90-76107ff7a9c3.png">

Once we can properly direct to the inner page or have a clear landing page, we can add a link here.